semcore/utils/src/uniqueID.ts

```diff
@@ -19,13 +19,15 @@ const createSource = (prefix = 'ui-kit-'): ContextType => ({ value: 1, prefix })
 
 export const useUID = (prefix?: string): string => {
   const context = register.get<ContextType>('uid-context', createSource(prefix));
-  const [uid] = React.useState<number>(context.value++);
+  const [uid] = React.useId ? [React.useId()] : React.useState<number>(context.value++);
+  const trimmedUid =
+    String(uid).startsWith(':') && String(uid).endsWith(':') ? String(uid).slice(1, -1) : uid;
```

Comment:
What if `React.useId()` will add some another incorrect symbol? Could we use `CSS.escape()` on server-side?

Reply:
React useId always adds `:` to make it unusable as css selector.

> Could we use CSS.escape() on server-side?

I would prefer to keep simple slice instead